### PR TITLE
fix the table element displaying response

### PR DESCRIPTION
**Pull request by XiChenn**

### CODE REVIEW

File: pybossa/util.py [hunk 1]
```diff
@@ -1253,6 +1253,12 @@ def process_table_component(tp_code, user_response):
     for table_element in table_elements:
         response_key = table_element.get('name')
         response_value = user_response.get(response_key, [])
+
+        # In case the response_value is dict, we need to convert it to a list
+        # so that the table-element can display the data correctly
+        if type(response_value) is dict:
+            response_value = list(response_value.values())
```
Comment (dchhabda) [inline pybossa/util.py:1260]: Is it guaranteed that response values are in the order in which table headers are ordered or the code need to build the list in the order the table headers are placed ensuring each data is listed in their respective header columns?

Comment (XiChenn) [reply pybossa/util.py:1260]: Great question! The order can be guaranteed as "Changed in version 3.7: Dictionary order is guaranteed to be insertion order." (source [here](https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects))

Comment (dchhabda) [reply pybossa/util.py:1260]: Thanks for confirming.

**Review by dchhabda — APPROVED** [review]
PR can be merged upon tests pass. Thanks!

**Pull request merged**